### DISCUSSION TITLE
How to draw a dinamic line with animation transforn?

### DISCUSSION THREAD

**Discussion by ramone02** [visgl/luma.gl#1734] [General]
Hi i need to draw a line in deck.gl, the line origin i need to set fixed in map, and end line rotate in 360°. thanks!

**Comment by ibgreen** [comment]
> Hi i need to draw a line in deck.gl

Why not just use the deck.gl LineLayer. 

> the line origin i need to set fixed in map

luma.gl doesn't have geospatial projection for shaders, deck.gl defines its own `project` module.
